themes-default/slim/views/editShow.mako

```diff
@@ -16,7 +16,6 @@ const startVue = () => {
                 seriesSlug: $('#series-slug').attr('value'),
                 seriesId: $('#series-id').attr('value'),
                 indexerName: $('#indexer-name').attr('value'),
-                config: MEDUSA.config,
```

Comment:
What? Why? It's being used.
https://github.com/pymedusa/Medusa/blob/b8207ce7ec20efa971bbb998933b00ca61c5e61b/themes-default/slim/views/editShow.mako#L142-L145

Unless you meant to replace it with the store config...